server/hotrod/src/main/scala/org/infinispan/server/hotrod/CacheDecodeContext.scala

```diff
@@ -32,7 +31,7 @@ class CacheDecodeContext(server: HotRodServer) extends ServerConstants with Log
    type BytesResponse = Bytes => Response
 
    val isTrace = isTraceEnabled
-   val SecondsInAMonth = 60 * 60 * 24 * 30
+   val MillisInAMonth = 60 * 60 * 24 * 30 * 1000L
```

Comment:
I see understand why we call it this, but maybe this should be just MillisIn30Days instead.  When I first saw this I had flash backs to DB2 date diff approximations [1]

[1] http://www-01.ibm.com/support/knowledgecenter/SSEPGG_9.7.0/com.ibm.db2.luw.sql.ref.doc/doc/r0000861.html?cp=SSEPGG_9.7.0%2F2-10-3-2-155